### DISCUSSION TITLE
Icon picker in Developer Tools

### DISCUSSION THREAD

**Discussion by wojtek14a** [home-assistant/frontend#10957] [Dashboard]
I'd like to propose **adding** the beautiful **Icon Picker** implemented in 2021.11 **to the Developer Tools as a new tab**.

Some configuration schemes still require providing the name of the icon, both while configuring entities in yaml as well as Lovelace cards. To browse through all possibilities we were forced to go to the MDI website (cheering for the new website which afaik is WIP!) but now we can use the more elegant picker. But in order to use it now, we have to for example select an entity (or open Dashboard edit mode and select card), go into edit mode, search for the icon, press cancel, go out of edit mode. 

It would be easier and quicker to open Developer Tools and go directly to icon picker (sometimes while working on config you already are in Developer Tools e.g. in Template).

What do you think about it? Is this a necessary tool or just my fantasy? Would it be difficult to implement it now, when the icon picker already exists?

P. S. While writing it I thought about using ~~Command Pallete~~ _Quick Bar_ for this with another symbol, like the `>` for command, maybe this is the way to go? Or both?

**Comment by michal7778** [comment]
Very good idea, especially Quick Bar for icons would be very convenient.

**Comment by saschabrockel** [comment]
I still wonder why this has no attention. I have multiple integrations with icons and never know which ones I have or not. It would be a game-changer.
